tests/framework-tests/core/js/ModelTransformationTests.js

```diff
@@ -2611,6 +2611,48 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         });
     });
 
+    /* --------------- fluid.transforms.indexOf tests -------------------- */
+
+    var indexOfTests = [{
+        message: "indexOf() should return the index of the value on the array.",
+        transform: {
+            type: "fluid.transforms.indexOf",
+            array: ["sheep", "dog"],
+            value: "dog"
+        },
+        expected: 1
+    }, {
+        message: "indexOf() should return -1 when the value is not found in the array.",
+        transform: {
+            type: "fluid.transforms.indexOf",
+            inputPath: ["sheep", "dog"],
+            value: "cat"
+        },
+        expected: -1
+    }, {
+        message: "indexOf() should return -1 when the value of the \"array\" argument is not an array type.",
+        transform: {
+            type: "fluid.transforms.indexOf",
+            inputPath: "sheep",
```

Comment:
This should read "array" rather than "inputPath", and when this is corrected you will find the test will fail through returning 0  ....